src/core/index.js

```diff
@@ -53,6 +53,8 @@ export default function SwaggerUI(opts) {
     showExtensions: false,
     showCommonExtensions: false,
     withCredentials: undefined,
+    hierarchicalTags: false,
+    tagSplitterChar: /[:|]/,
```

Comment:
Is regex really needed here? Why not use a delimiter like "|"

Reply:
Since `match` accepts both string and regex, I figured it would be useful to make the default value a regex searching for various possible separator chars. That way you don't need to worry about supplying additional config if you've used _any_ of the acceptable default chars.

Reply:
Nice